src/Helper/Dsn.php

```diff
@@ -56,9 +56,4 @@ public function getDsn(): string
 
         return $dsn;
     }
-
-    public function getDriver(): string
```

Comment:
Why removing `getDriver()`?

Reply:
1. Not used anywhere.
2. It is necessary either to make all properties available, or none.

Reply:
> It is necessary either to make all properties available, or none.

Why is it necessary?

Reply:
> Why is it necessary?

All other things being equal, making a getter for only one property is discrimination :)